Strapi 5 templates (#2192)
strapi/documentation: docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/templates.md

---

> title: Templates are now typical Strapi applications

Author: ```suggestion
title: Templates are now regular, standalone Strapi applications
```

> description: Strapi 5 has strict requirements on the configuration filenames allowed to be loaded.

Author: ```suggestion
description: Templates have been fully rewritten in Strapi 5 and now are standalone, regular Strapi applications, making it easier to create, distribute, and re-use them.
```

displayed_sidebar: devDocsMigrationV5Sidebar
tags:
 - breaking changes
 - templates
 - upgrade to Strapi 5
---

import Intro from '/docs/snippets/breaking-change-page-intro.md'
import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
import NoPlugins from '/docs/snippets/breaking-change-not-affecting-plugins.md'
import NoCodemods from '/docs/snippets/breaking-change-not-handled-by-codemod.md'

# Templates are now standalone, regular Strapi applications

> Templates have been fully rewritten in Strapi 5 and now are standalone, regular Strapi applications, making it much easier to create, distribute, and re-use them.

Author: ```suggestion
Templates have been fully rewritten in Strapi 5 and now are standalone, regular Strapi applications, making it easier to create, distribute, and reuse them.
```
